Makefile

```
@@ -18,3 +21,7 @@ e2e: build
 .PHONY: build
 build:
```

Lets run generate for every build to keep it sync with the code

```suggestion
build: generate
```

---

fixed